src/util/fs.ts

```diff
@@ -15,8 +15,9 @@ import { promisify } from 'util';
 import { SfdxError } from '../sfdxError';
 
 type PerformFunction = (filePath: string, file?: string, dir?: string) => Promise<void>;
+type PerformFunctionSync = (filePath: string, file?: string, dir?: string) => void;
 
-export const fs = {
+export const fs = Object.assign({}, fsLib, {
```

Comment:
This is what I was talking about @WillieRuemmele @nramyasri-sf - now all fs-graceful methods will be on the fs object.